core/src/main/java/org/apache/calcite/sql/validate/SqlValidatorUtil.java

```diff
@@ -1309,6 +1311,64 @@ public static boolean isMeasure(SqlNode selectItem) {
     return null;
   }
 
+  /**
+   * When the array element does not equal with the array component type, making explicit casting.
```

Comment:
i think you can remove `with` and change making->make

Reply:
thanks. fixed.